### PR TITLE
SCRD-3481 - add networks to server details modal

### PR DESCRIPTION
**Pull request by jeremy-moffitt**

### CODE REVIEW

**Comment by eapodaca** [comment]
Code looks reasonable enough. Layout wise it might be nice to make the 'network' divs the width ~50% the width of the modal? they look kinda funky all different widths.

**Comment by jeremy-moffitt** [comment]
made the network divs 50% minus the margin, fits 2 per line and they're all the same width

File: src/components/CollapsibleTable.js [hunk 1]
```diff
@@ -127,18 +127,65 @@ class CollapsibleTable extends Component {
     );
   }
 
+  /**
+   * renders the view server action during the Day 0 mode of the install,
+   * this function is not called during Day2 operations due to changes in how
+   * the table handles actions after install (see getContextMenuItems)
+   */
```
Comment (GarySmith) [inline src/components/CollapsibleTable.js]: This comment seems backward; it only seems to show up in day 2

File: src/components/CollapsibleTable.js [hunk 2]
```diff
@@ -127,18 +127,65 @@ class CollapsibleTable extends Component {
...
+   * since loading them for every server would be intensive
+   */
+  loadServerDetails = (server, retries) => {
+    //the internalModel is not necessarily loaded right away,
+    //check again after 1 second if not already loaded
+    if(this.props.internalModel !== undefined) {
+      let internalModelServers = this.props.internalModel.getIn(['internal', 'servers']).toJS();
+      let fullModelServer = internalModelServers.find(s => s.id == server.id);
+      let server_networks_list = [];
+      for (let inet in fullModelServer.interfaces) {
```
Comment (GarySmith) [inline src/components/CollapsibleTable.js]: If you revisit this, consider `Object.values(fullModelServer.interfaces)` .

**Pull request merged**